design/001-RPC-Interface-Specification.md

```diff
@@ -79,13 +77,17 @@ The following 3 data formats will be used in the APIs.
     * `Currency`: [String] Currency used for specifying the amount.
     * `Balance`: [Map of `Alias` to `Balance`]
       * `Alias`: [String] `Self` is a special alias for the user of the node.
-      * `Balances`: [String] Amount held by the `Peer` corresponding to the Alias.
+      * `Balances`: [String] Balance of the `Peer` corresponding to the Alias.
 
-3. `Payment Channel`
+3. `Payment Channel Info`
 
     * `Channel ID`: [String] Unique ID of the channel.
     * `BalanceInfo`: [Balance Info]
     * `Version`: [String] Current Version of the channel.
```

Comment:
What's versino? Version of the software? Do you expect the node or session to support different versions of the protocol? Otherwise version query could be moved to either node or session API?

Reply:
It is the version of the state channel, which is incremented each time when the state is updated.

Reply:
Since the term 'version' makes confusion with other versions like software version, protocol version etc, may be we can use the term 'stateCounter' instead of 'version' which actually increments the counter for each updation of state.

Reply:
Okay, now i get the point. I agree `StateCounter` is a more clear term than `Version`. Will update the spec.

Reply:
Well, if the term `Version` is used for this field in go-perun, it may be better to keep in sync with the terminology for the time being. In that case I'd suggest to add a description of what it means.

Reply:
Okay, I have retained the term `Version` and have added additional description to it.